README.md

```diff
@@ -68,7 +68,13 @@ Relevant Waku resources related to the specifications located in this repository
 Contributions are welcome from any party. 
 Contributors can create specifications relating to the Waku domain and
 create a pull request to begin discussion.
-The recommended [template](./template.md) may be used for new proposed specifications.
+
+Please adhere to the following contribution guidelines:
+- use the recommended [template](./template.md) for new proposed specifications
+- use keywords as per the language recommendations in the [template](./template.md) and [Vac COSS](https://github.com/vacp2p/rfc-index/blob/a5b24ac0a27da361312260f9da372a0e6e812212/vac/1/coss.md#language)
+- use [semantic breaks](https://sembr.org/)
+- links to other Waku, Vac or related specifications must be to the corresponding Github repository and not to a webpage.
```

Comment:
Should this be org related specifications because some specifications not apart of IFT are referenced to a webpage.

Reply:
Good point. I've clarified the phrasing now.